src/mono/wasm/debugger/tests/debugger-test/debugger-get-properties-test.cs

```diff
@@ -115,6 +115,103 @@ public async Task InstanceMethodAsync()
         }
     }
 
+    public class BaseBaseClass2
+    {
+        private string _baseBase_BackingFieldForAutoProperty = "BaseBase#BaseBase_BackingFieldForAutoProperty";
```

Comment:
The backing field that we mean in the code is one autogenerated by the compiler, like when we define a property as:
`bool AutoProp { get; set; }`

.. for which the compiler will generate field with `k__BackingField` in the name.